src/Init/Tactics.lean

```diff
@@ -941,6 +1008,61 @@ If there are several with the same priority, it is uses the "most recent one". E
 ```
 -/
 syntax (name := simp) "simp" (Tactic.simpPre <|> Tactic.simpPost)? (ppSpace prio)? : attr
+
+
+/-- The possible `norm_cast` kinds: `elim`, `move`, or `squash`. -/
+syntax normCastLabel := &"elim" <|> &"move" <|> &"squash"
+
+/--
+The `norm_cast` attribute should be given to lemmas that describe the
+behaviour of a coercion in regard to an operator, a relation, or a particular
```

Comment:
```suggestion
behaviour of a coercion with respect to an operator, a relation, or a particular
```
This phrasing sounds more natural in my ears, but this might just be my North American background. Feel free to reject if that's the case.